canonical-web-and-design/ubuntu.com
# Update TCO calculator contact form

## Done

- Remove old `tco-calculator.html` as it's been replaced
- Update [contact us](https://docs.google.com/document/d/1n3ExZwFVlw5LyAp7O3CgPthVgx8yRX5-Wg_gBQLf73c/edit?disco=AAAAOSt-wZQ) to have only contact information inputs visible to the user, but:
  - have default values for all the previous [fields](https://docs.google.com/document/d/1n3ExZwFVlw5LyAp7O3CgPthVgx8yRX5-Wg_gBQLf73c/edit#heading=h.q4y1ndq4o59i) 
  - If the user has updated the input fields on the page then use these values instead
- Pass these hidden values to Marketo 

## QA

- View page at: https://ubuntu-com-10527.demos.haus/openstack/pricing-calculator
1.  Click "Get in touch"
2.  Check the default data populates the comment by checking `Comments_from_lead__c.value` in the console returns all the fields and default values
3.  Now change one of the inputs e.g. vCPUs or Number of instances
4. Repeat step 2 and check the updated value is seen in the comment.
5. Fill in the form with test data to check it submits ok


## Issue / Card

Fixes [#4430](https://github.com/canonical-web-and-design/web-squad/issues/4430)
